tests/ops/functions/conftest.py

```diff
@@ -44,6 +44,7 @@
     qml.adjoint(qml.RX(1.1, 0)),
     Tensor(qml.PauliX(0), qml.PauliX(1)),
     qml.Hamiltonian([1.1, 2.2], [qml.PauliX(0), qml.PauliZ(0)]),
```

Comment:
Heads up that @lillian542  pushed a fix for this to `ham-tests` that converts this to a legacy Hamiltonian.